core/os/android/adb/device.go

```diff
@@ -145,6 +145,11 @@ func newDevice(ctx context.Context, serial string, status bind.Status) (*binding
 		d.To.Configuration.OS.Build = strings.TrimSpace(description)
 	}
 
+	// Query device Perfetto service state
+	if perfettoCapability, err := d.QueryPerfettoServiceState(ctx); err == nil {
```

Comment:
This drops the error. We should at least log it.

Reply:
Wait, but the errors above are all dropped?

Reply:
Alright. OK.